lib/active_admin/base_controller.rb

```diff
@@ -73,5 +78,10 @@ def determine_active_admin_layout
       ACTIVE_ADMIN_ACTIONS.include?(params[:action].to_sym) ? false : 'active_admin'
     end
 
+    def active_admin_root
+      controller, action = active_admin_namespace.root_to.split '#'
+      {controller: controller, action: action}
+    end
+
```

Comment:
You assume this:
```ruby
active_admin_namespace.root_to = "dashboard#show"
```
But this can also happen:
```ruby
active_admin_namespace.root_to = redirect_to("...")
# or
active_admin_namespace.root_to = -> { ... }
```

But today I have see that the old version did the same, so it's ok.